### PR TITLE
#DP-1352 feat: added bulk approve api's  documentation

### PR DESCRIPTION
**Pull request by santoshilimi**

### CODE REVIEW

**Comment by keshavprasadms** [comment]
Hello @santoshilimi,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://634.qa.docs.sunbird.org/api-documentation/ .<br/> **Note:** Only _api-documentation_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://634.qa.docs.sunbird.org/api-documentation/apis/bulkapproveapi// <br /> - http://634.qa.docs.sunbird.org/api-documentation/apis/bulkapproveapi// <br />  <br/> **Modified sections are -** <br />

File: apis/bulkapproveapi/bulkapproveapi.yaml [hunk 1]
```diff
@@ -0,0 +1,670 @@
+openapi: 3.0.0
+info:
+  title: Bulk_Approve
+  termsOfService: 'https://github.com/project-sunbird/sunbird-commons/blob/master/LICENSE'
+  description: These api's are all about the bulk approval of the contents and other scenarios included in that.
```
Comment (Sunitashelar) [inline apis/bulkapproveapi/bulkapproveapi.yaml:5]: @santoshilimi can you clarify what you mean by 'other scenarios included in that'?

**Review by Sunitashelar — CHANGES_REQUESTED** [review]

File: apis/bulkapproveapi/bulkapproveapi.yaml (hunk 1 above)
Comment (santoshilimi) [reply apis/bulkapproveapi/bulkapproveapi.yaml]: Please check,

**Comment by keshavprasadms** [comment]
Hello @santoshilimi,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://634.qa.docs.sunbird.org/api-documentation/ .<br/> **Note:** Only _api-documentation_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://634.qa.docs.sunbird.org/api-documentation/apis/bulkapproveapi// <br /> - http://634.qa.docs.sunbird.org/api-documentation/apis/bulkapproveapi// <br />  <br/> **Modified sections are -** <br />

**Review by Sunitashelar — APPROVED** [review]

**Pull request merged**